index-import.cpp

```diff
@@ -242,7 +225,30 @@ static bool cloneRecords(StringRef recordsDirectory,
 int main(int argc, char **argv) {
   cl::ParseCommandLineOptions(argc, argv);
 
-  auto remapper = Remapper::createFromCommandLine(PathRemaps);
+  Remapper remapper;
+  // Parse the the path remapping command line flags. This converts strings of
+  // "X=Y" into a (regex, string) pair. Another way of looking at it: each
+  // remap is equivalent to the s/pattern/replacement/ operator.
+  auto errors = 0;
```

Comment:
This can be a `bool` instead of a counter.

Reply:
Yeah, I had more diffs and in my reshuffle due to clang import dropped some... added now.